### PR TITLE
#808: render2d: equilibrium arrows rendering supported

### PR DESCRIPTION
**Pull request by even1024**

### CODE REVIEW

File: core/render2d/render_context.h [hunk 1]
```diff
@@ -32,6 +32,12 @@ namespace indigo
     class RenderContext
     {
     public:
+        enum
+        {
+            EOpenArrow,
+            EBowArray,
+            ETriangleArrow
+        };
```
Comment (mkviatkovskii) [inline core/render2d/render_context.h:40]: enum class

**Comment by mkviatkovskii** [comment]
@even1024 please rename the pull request so it's clear to understand what parts of the code changed and for what reasons. Like:
```
#808: render2d: equilibrium arrows rendering supported
```

**Comment by even1024** [comment]
> @even1024 please rename the pull request so it's clear to understand what parts of the code changed and for what reasons. Like:
> 
> ```
> #808: render2d: equilibrium arrows rendering supported
> ```
Fixed

**Comment by mkviatkovskii** [comment]
> > @even1024 please rename the pull request so it's clear to understand what parts of the code changed and for what reasons. Like:
> > ```
> > #808: render2d: equilibrium arrows rendering supported
> > ```
> 
> Fixed

Not exactly, we don't need feature prefix, but need `#`, because it's used by GitHub to find issue references and mark them

**Review by mkviatkovskii — APPROVED** [review]

**Pull request merged**